pkg/webui/components/header-v2/header-v2.styl

```
@@ -0,0 +1,50 @@
+// Copyright © 2023 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+.container
+  padding: 0 $cs.l
```

Don't use these values arbitrarily. The correct variable here is $cs.m = 1rem. Please make sure to double-check any variable that you use by comparing the values.
```suggestion
  padding: 0 $cs.m
```